openfeature/api.py

```diff
@@ -26,12 +30,19 @@
     "shutdown",
     "add_handler",
     "remove_handler",
+    "set_transaction_context_propagator",
+    "set_transaction_context",
+    "get_transaction_context",
 ]
 
 _evaluation_context = EvaluationContext()
 
 _hooks: typing.List[Hook] = []
 
+_transaction_context_propagator: TransactionContextPropagator = (
+    NoopTransactionContextPropagator()
```

Comment:
Should the SDK initialize a functional transaction context propagator by default?

Reply:
We do something similar to this in JS. What would the alternative be?

Reply:
Having a default implementation that is not a no-op. But this could introduce a performance penalty if you are not using it. Probably better to default to no-op.